### PR TITLE
bpo-32436: Document PEP 567 changes to asyncio.

### PR DESCRIPTION
**Pull request by 1st1**
https://bugs.python.org/issue32436

### CODE REVIEW

**Comment by 1st1** [comment]
@elprans please take a look.

File: Doc/library/asyncio-eventloop.rst [hunk 1]
```diff
@@ -137,19 +137,31 @@ keywords to your callback, use :func:`functools.partial`. For example,
    Any positional arguments after the callback will be passed to the
    callback when it is called.
 
+   An optional keyword-only *context* parameter allows to specify a custom
```
Comment (elprans) [inline Doc/library/asyncio-eventloop.rst]: The optional *context* keyword-only argument

File: Doc/library/asyncio-eventloop.rst [hunk 2]
```diff
@@ -182,10 +194,18 @@ a different clock than :func:`time.time`.
    is called. If you want the callback to be called with some named
    arguments, use a closure or :func:`functools.partial`.
 
+   An optional keyword-only *context* parameter allows to specify a custom
```
Comment (elprans) [inline Doc/library/asyncio-eventloop.rst]: The optional context keyword-only argument

File: Doc/library/asyncio-task.rst [hunk 3]
```diff
@@ -275,19 +275,27 @@ Future
...
+   .. method:: add_done_callback(callback, *, context=None)
 
       Add a callback to be run when the future becomes done.
 
-      The callback is called with a single argument - the future object. If the
+      The *callback* is called with a single argument - the future object. If the
       future is already done when this is called, the callback is scheduled
       with :meth:`~AbstractEventLoop.call_soon`.
 
+      An optional keyword-only *context* parameter allows to specify a custom
```
Comment (elprans) [inline Doc/library/asyncio-task.rst]: The optional context keyword-only argument

File: Doc/whatsnew/3.7.rst [hunk 4]
```diff
@@ -608,6 +608,17 @@ include:
   destroying the event loop.
   (Contributed by Yury Selivanov in :issue:`32314`.)
 
+* asyncio gains support for :mod:`contextvars`.
```
Comment (elprans) [inline Doc/whatsnew/3.7.rst]: s/gains/gained/

**Review by elprans — APPROVED** [review]

File: Doc/library/asyncio-eventloop.rst (hunk 1 above)
Comment (1st1) [reply Doc/library/asyncio-eventloop.rst]: Will fix paremeter to argument

**Pull request merged**